### DISCUSSION TITLE
Welcome to Whatsapp-Bot-AI Discussions!

### DISCUSSION THREAD

**Discussion by adityakumar48** [adityakumar48/Whatsapp-Bot-AI#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by emadammar** [comment]
Where is the location of the WhatsApp phone number?

**Reply by adityakumar48** [reply]
You need to set up and scan the code

**Comment by add146** [comment]
Hi, cool app, I really enjoy with friends in whatsapp group :)
The installation is very easy even I don't have a programming background,
there are a few things I ask:

1. Why does it always scan device every time the connection is lost, whereas in other applications you just have to run the program again

2. Is there an alternative mongodb, because using share account is often disconnected

3. Can't make stickers from videos

thank you for attention

**Reply by adityakumar48** [reply]
Thanks for your appreciation 
Answers to your Questions

1 - Have you done a proper .env file setup? (Mongodb URI) because the session is stored in your cloud db so every time get it from the database no need to scan it again.

2 - I don't know much about different DBs, I just followed the documentation, check out maybe that is useful.

3 - yeah because I don't integrate that feature it just converts the image to a sticker.

**Reply by add146** [reply]
Thanks for reply

1 - Yes i have insert string _mongodb+srv://...._
i think is connected, it say

MongoDB database connection established successfully

<QR code image>

3 - everytime using video for sticker app becomes disconnected :(

**Reply by adityakumar48** [reply]
1 - maybe there is a package issue
 
3 - its not compatible for video to sticker
